src/serialization/compact/CompactStreamSerializer.ts

```diff
@@ -138,6 +139,9 @@ export class CompactStreamSerializer {
 
     writeObject(output: PositionalObjectDataOutput, obj: any) : void {
         const compactSerializer = this.classToSerializerMap.get(obj.constructor);
+        if (compactSerializer == undefined) {
+            throw new HazelcastSerializationError(`No serializer is registered for class ${obj.constructor.name}.`)
```

Comment:
I think we can do the following change since constructor functions can be used with compact:
```suggestion
            throw new HazelcastSerializationError(`No serializer is registered for class/constructor ${obj.constructor.name}.`)
```

Reply:
Fixed at https://github.com/hazelcast/hazelcast-nodejs-client/pull/1365/commits/3345e5697bb3247845774bd13ae536ebe3124fc2.